<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fight</class>
 <widget class="QWidget" name="Fight">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="img_label1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>181</width>
     <height>281</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="info_label1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>191</width>
     <height>231</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="img_label2">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>50</y>
     <width>181</width>
     <height>281</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="info_label2">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>360</y>
     <width>191</width>
     <height>231</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pre_btn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>640</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>上一个</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next_btn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>640</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>下一个</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>请选择你的出战精灵</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fight_btn">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>20</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>开始战斗</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="fightInfo">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>80</y>
     <width>311</width>
     <height>511</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>309</width>
      <height>509</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
